lib/std/mem.zig

```diff
@@ -2903,9 +2903,10 @@ pub fn WindowIterator(comptime T: type) type {
 
         const Self = @This();
 
-        /// Returns a slice of the first window. This never fails.
+        /// Returns a slice of the first window.
         /// Call this only to get the first window and then use `next` to get
         /// all subsequent windows.
+        /// If called after iteration has begun, it will panic.
```

Comment:
an assertion doesnt inherently mean a panic occurs, its just a case of [safety-checked undefined behavior](https://ziglang.org/documentation/master/#Undefined-Behavior). i think this (and the others) should say something more along the lines of `Asserts that iteration has not begun`, there are many cases of such phrasing across the stdlib

Reply:
you're absolutely right, I will change it to follow the docs convention for these cases